common/protocol/Makefile.mk

```diff
@@ -21,8 +21,8 @@ common_protocol_libolaproto_la_CXXFLAGS = $(COMMON_CXXFLAGS_ONLY_WARNINGS)
 
 common_libolacommon_la_LIBADD += common/protocol/libolaproto.la
 
-common/protocol/Ola.pb.cc common/protocol/Ola.pb.h: common/protocol/Ola.proto
+common/protocol/Ola.pb.cc common/protocol/Ola.pb.h: common/protocol/Makefile.mk common/protocol/Ola.proto
```

Comment:
I started investigating this because include/ola/plugin_id.h kept getting rebuilt while tweaking the doxygen files stuff.

I can confirm it only gets rebuilt if the files listed are touched and not if Makefile is.